### PR TITLE
[API] Demo of REST API test automation

### PR DESCRIPTION
**Pull request by mialeska**

### CODE REVIEW

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿@API @demo
+Feature: Demo API
+
+Scenario: GET user info
+	When I send GET '/users/aqualityAutomation' request to github and save the 'response'
+	Then the status code of the 'response' is '200'
```
Comment (n-verbitsky) [inline Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature]: Could you review the indents here?

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        And the 'response' matches json schema 'Users/UserResponse'
+        And the 'login' is 'aqualityAutomation' in the 'response'
+        And the 'type' is 'User' in the 'response'
+    When I save the user from the 'response' as 'user1'
```
Comment (n-verbitsky) [inline Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature:10]: Usually `When` step should not follow `Then` steps. Consider making it `Then`

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        And I send GET '/users/aquality-automation' request to github and save the 'response2'
```
Comment (n-verbitsky) [inline Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature]: `and` is reserved keyword. Consider splitting in two steps

**Review by n-verbitsky — APPROVED** [review]

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature (hunk 2 above)
Comment (mialeska) [reply Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature:10]: Cannot agree here, we can use When/Then pair several times in complex scenarios. We can follow Demo.feature as example

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature (hunk 3 above)
Comment (mialeska) [reply Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature]: reworded the step, splitting is not possible here

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature (hunk 1 above)
Comment (mialeska) [reply Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Features/DemoApi.feature]: thanks, fixed

**Pull request merged**